data/lang/module-donatetocranks/en.json

```
@@ -71,6 +71,54 @@
     "description": "",
     "message": "Liberate your ship! Please choose amount to donate. Happy hacking."
   },
+  "FLAVOUR_6_ADTITLE": {
+    "description": "",
+    "message": "VETERAN DAY!"
+  },
+  "FLAVOUR_6_DESC": {
+    "description": "",
+    "message": "Support the veterans of the {MILITARY}."
+  },
+  "FLAVOUR_6_MESSAGE": {
+    "description": "",
+    "message": "The {MILITARY}. You can always count on them.\nCan they count on you?"
+  },
+  "FLAVOUR_7_ADTITLE": {
+    "description": "",
+    "message": "DO THEY HAVE YOUR SUPPORT?"
+  },
+  "FLAVOUR_7_DESC": {
+    "description": "",
+    "message": "Support the veterans of the {POLICE}."
+  },
+  "FLAVOUR_7_MESSAGE": {
+    "description": "",
+    "message": "Those brave men and women of the {POLICE} who have dedicated their lives to keeping {SYSTEM} safe!"
```

This isn't a sentence as written, there are only linking verbs and no primary verb. I've added a call to action here, feel free to tweak it if you think it could be better.

```suggestion
    "message": "Those brave men and women of the {POLICE} who have dedicated their lives to keeping {SYSTEM} safe need your support!"
```